### ISSUE TITLE
[Settings Startup Page]: Click command does not work for the 'Launch size' expand and collapse buttons.

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/terminal#14391]
### Windows Terminal version

1.17.3052.0

### Windows build number

Windows 11 Version 22H2 (OS Build 25206.1000)

### Steps to reproduce

1. Open Windows Terminal.
2. Navigate to drop down button by 'Ctrl+Shift+Space' key.
3. Navigate to 'Settings' button by down arrow key and select it.
4. Open Voice Access by 'Settings > Accessibility > Speech > Turn on Voice Access.
5. Give command as 'Click' by Voice Access on the page to access 'Launch size'.
6. Observe the issue.

**User Experience**: -
Voice Access users will not be able to access the buttons with click command, hence will an impression to users that the name of control is not properly defined.

**Guideline Reference**:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01

**Attachment**s:
[Click command does not work for the 'Launch size' expand and collapse buttons..zip](https://github.com/microsoft/terminal/files/10010024/Click.command.does.not.work.for.the.Launch.size.expand.and.collapse.buttons.zip)

### Expected Behavior

Click command should work for activating an interactive control irrespective of the nature of the control.

Refer this link: [Voice access – Commands List (office.com)](https://sway.office.com/Q5JvoIGBI5glyoKw?ref=Link)

### Actual Behavior

Click command does not work for the 'Launch size' expand and collapse buttons. Expand and collapse command works for the button.

**NOTE:**
This happens for all similar button controls throughout the application.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
